data/avro-schema/build.gradle

```diff
@@ -7,6 +7,7 @@ plugins {
     id 'corda.common-publishing'
     id 'corda.java-only'
     id "com.github.davidmc24.gradle.plugin.avro-base"
+    id 'corda.api-scanner'
```

Comment:
I don't think this project needs to be includes as the AVRO schemas themselves are the public API.